easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -654,7 +619,7 @@ def _generate_template_values(self, ignore=None, skip_lower=True):
             if v is None:
                 del self.template_values[k]
 
-    @handle_deprecated_easyconfig_parameter
+    @check_replaced_easyconfig_parameter
```

Comment:
renamed/fixed the decorator as `handle_deprecated_or_replaced_easyconfig_parameters`